injected/src/features/autofill-password-import.js

```diff
@@ -76,39 +99,116 @@ export default class AutofillPasswordImport extends ContentFeature {
                 ? {
                     style: this.signInButtonStyle,
                     element,
-                    shouldTap: this.#signInButtonSettings?.shouldAutotap ?? false
+                    shouldTap: this.#signInButtonSettings?.shouldAutotap ?? false,
+                    shouldWatchForRemoval: false
                 }
                 : null
         } else {
             return null
         }
     }
 
+    /**
+     * Removes the overlay if it exists.
+     */
+    removeOverlayIfNeeded () {
+        const existingOverlay = document.getElementById(OVERLAY_ID)
+        if (existingOverlay != null) {
+            existingOverlay.style.display = 'none'
+            existingOverlay.remove()
+        }
+    }
+
+    /**
+     * Inserts an overlay element to animate, by adding a div to the body 
+     * and styling it based on the found element. 
+     * @param {HTMLElement|Element} mainElement 
+     * @param {any} style 
+     * @returns {HTMLElement|Element|null}
+     */
+    insertOverlayElement (mainElement, style) {
+        this.removeOverlayIfNeeded()
+
+        const overlay = document.createElement('div')
+        overlay.setAttribute('id', OVERLAY_ID)
+        const svgElement = mainElement.parentNode?.querySelector('svg') ?? mainElement.querySelector('svg')
+
+        const isRound = style.borderRadius === '100%'
```

Comment:
Should we be using `getComputedStyle` for this?

Reply:
@jonathanKingston this is just the "preset style" that _we're_ setting here, if you're talking about the line mentioned here. That `style` parameter gets passed on to this function.

Reply:
Ah sorry misread thanks for the clarification. :)